src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -85,6 +118,41 @@ New Relic AI is becoming more accurate and responsive the more it's used. We wan
 
 Also, New Relic AI can recognize multiple languages and respond in them. If you have use cases for multi-lingual support, please test them with our AI and give us feedback.
 
+New Relic AI responses are AI-generated suggestions. While efforts are made to keep information and responses up to date, suggestions may not always be accurate, complete, or error-free. We would like to encourage you to have a human verify any information provided by our AI using other sources before any action is taken. AI's suggestions are provided without any warranties or guarantees of accuracy.
 
+## Sample questions [#sample-questions] 
```

Comment:
@homelessbirds what would you think of moving this section so it follows the section about where you can interact with NRAI? It seems like readers are going to mostly be focused on the usage and not the security and how AI continues to build accuracy.